languages/erlang/exercises/concept/basics/.docs/introduction.md

```diff
@@ -0,0 +1,60 @@
+To define a variable and bind a value to it, use the `=` operator:
+
+```erlang
+MyValue = 10.
+```
+
+Variable names must begin with an uppercase letter and are generally written in camel case.
+
+To enforce [immutabilty][immutability], once a variable is bound to a value, it cannot be updated:
```

Comment:
There is a link to a resource here but I don't think that link is declared anywhere. As per [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsintroductionmd):

> Links should be used sparingly, if at all. While a link explaining a complex topic like recursion might be useful, for most concepts the links will provide more information than needed so explaining things concisely inline should be the aim.

In this case, I think it is fine to mention immutability (maybe using underscores to make it italic), but not to link to an external resources. Linking to external resources is great for `after.md` documents though, so please do add it there.

Reply:
The above comment on not preferrably not using links in the introduction.md document also applies to the other links in this file.